terraform/variables.tf

```
@@ -39,4 +39,10 @@ variable "units" {
   description = "Number of units"
   type        = number
   default     = 1
-}
\ No newline at end of file
+}
+
+variable "grafana_data-storage-size" {
```

Is there a reason you decided to not use underscores in the var name?
```suggestion
variable "grafana_data_storage_size" {
```

---

nope, just a typo.